src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics.CodeAnalysis;
```

Comment:
@daxian-dbw, your last commit had 4 failures in `PowerShell-CI-linux`
Verify approved aliases list.All approved aliases present (no new aliases added, no aliases removed)
```powershell
Expected $null or empty, but got @(@{InputObject=gcb -> Get-Clipboard; SideIndicator==>}, @{InputObject=scb -> Set-Clipboard; SideIndicator==>}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 553
553:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @('Get-Clipboard', 'Set-Clipboard').
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 584
584:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Update-Help; ConfirmImpact=Medium; SideIndicator==>}, @{Name=Write-Warning; ConfirmImpact=None; SideIndicator==>}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```
Basic Job Tests.Wait-Job tests.Can timeout waiting for a job
```powershell
Cannot perform operation because object "Pipeline" has already been disposed.
Object name: 'Pipeline'.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Job/Jobs.Tests.ps1: line 139
139:             Get-Job | Where-Object { $_.Id -ne $startedJob.Id } | Remove-Job -ErrorAction SilentlyContinue -Force

```